docs/common_links.rst

```diff
@@ -175,6 +182,9 @@
    nitpick_ignore_regex in docs/conf.py so that it doesn't get counted
    as an error in a nitpicky doc build (e.g., tox -e doc_build_nitpicky).
 
+.. _`astropy.units`: https://docs.astropy.org/en/stable/units/index.html
+.. |astropy.units| replace:: `astropy.units`
+
```

Comment:
```suggestion
```

Since this replacement isn't used anywhere, it should be deferred to a PR that utilizes it.  Are we sure `` `astropy.units` `` does not already link to the appropriate documentation?  That is, do we actually need a replacement to accomplish the linking?